<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDefinedConstantsDialog</class>
 <widget class="QDialog" name="UserDefinedConstantsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User-Defined Constants</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources_editor.qrc">
    <normaloff>:/qtgameengine/icons/constants.png</normaloff>:/qtgameengine/icons/constants.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonInsert">
          <property name="toolTip">
           <string>Insert a constant before the current one</string>
          </property>
          <property name="text">
           <string>&amp;Insert</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/add.png</normaloff>:/qtgameengine/icons/add.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonSort">
          <property name="toolTip">
           <string>Sort the constants</string>
          </property>
          <property name="text">
           <string>&amp;Sort</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/sort.png</normaloff>:/qtgameengine/icons/sort.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonDelete">
          <property name="toolTip">
           <string>Delete the current constant</string>
          </property>
          <property name="text">
           <string>&amp;Delete</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/delete.png</normaloff>:/qtgameengine/icons/delete.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButtonLoad">
          <property name="toolTip">
           <string>Load the constants from a text file</string>
          </property>
          <property name="text">
           <string>&amp;Load</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/open.png</normaloff>:/qtgameengine/icons/open.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonUp">
          <property name="toolTip">
           <string>Moves the constant up</string>
          </property>
          <property name="text">
           <string>&amp;Up</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/arrow-up.png</normaloff>:/qtgameengine/icons/arrow-up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="toolTip">
           <string>Append a constant at the end of the list</string>
          </property>
          <property name="text">
           <string>&amp;Add</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/add.png</normaloff>:/qtgameengine/icons/add.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonClear">
          <property name="toolTip">
           <string>Clear all the constants</string>
          </property>
          <property name="text">
           <string>&amp;Clear</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/new.png</normaloff>:/qtgameengine/icons/new.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonDown">
          <property name="toolTip">
           <string>Moves the constant down</string>
          </property>
          <property name="text">
           <string>Do&amp;wn</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/arrow-down.png</normaloff>:/qtgameengine/icons/arrow-down.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButtonSave">
          <property name="toolTip">
           <string>Save the constants to a text file</string>
          </property>
          <property name="text">
           <string>Sa&amp;ve</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_editor.qrc">
            <normaloff>:/qtgameengine/icons/save.png</normaloff>:/qtgameengine/icons/save.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources_editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserDefinedConstantsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserDefinedConstantsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
